xtext-website/_posts/releasenotes/2024-11-25-version-2-37-0.md

```diff
@@ -14,6 +14,8 @@ As you might have recognized, the number of people contributing to Xtext on a re
 
 ## Relevant changes
 
+- `EcoreUtil2` now provides a variant of `getAllContentsOfType()` for Resources.
```

Comment:
It would be best if you could add the parameter types in `getAllContentsOfType`, e.g., `getAllContentsOfType(Resource, Class)`

Reply:
Done